lib/grape/endpoint.rb

```diff
@@ -234,6 +234,15 @@ def equals?(endpoint)
       (options == endpoint.options) && (inheritable_setting.to_hash == endpoint.inheritable_setting.to_hash)
     end
 
+    # the purpose of this override is solely for stripping internals when an error occurs while calling
+    # an endpoint through an api. See #https://github.com/ruby-grape/grape/issues/2398
+    # Otherwise, it calls super
+    def inspect
+      return super unless env
+
+      "#{self.class} in `#{route.origin}' endpoint"
```

Comment:
Was this meant to be quoted differently?

```suggestion
      "#{self.class} in '#{route.origin}' endpoint"
```

Reply:
https://bugs.ruby-lang.org/issues/16495
Looks like something cultural